app/workers/users/subscribe_to_mailchimp_newsletter_worker.rb

```diff
@@ -6,7 +6,7 @@ class SubscribeToMailchimpNewsletterWorker
     def perform(user_id)
       user = User.find_by(id: user_id)
 
-      MailchimpBot.new(user).upsert if user
+      Mailchimp::Bot.new(user).upsert if user
```

Comment:
Since this is called asynchronously, we can't really delete the old/original `MailchimpBot` in this PR because when we deploy this there could be workers in the queue still referencing it. I'll create a separate PR after this one is deployed to remove it.

Reply:
You could keep the file with empty methods hanging around. You could also stub the name somewhere :laughing: 
```
MailchimpBot = Mailchimp::Bot
```
Got the idea from sidekiq https://github.com/mperham/sidekiq/wiki/FAQ#how-do-i-safely-rename-a-worker